Do not bail on volumes smaller than 800K. It is possible to format 400K
floppies as HFS and it is possible to have HFS disk images of any size.
Many HFS disk images Apple distributes (such as System 6 installation
floppies) are smaller than 800K.
--- libhfs/volume.c.orig	1998-11-02 16:09:10.000000000 -0600
+++ libhfs/volume.c	2022-04-19 19:13:09.000000000 -0500
@@ -269,8 +269,6 @@
 	ERROR(EINVAL, "volume partition is empty");
     }
 
-  if (vol->vlen < 800 * (1024 >> HFS_BLOCKSZ_BITS))
-    ERROR(EINVAL, "volume is smaller than 800K");
 
   return 0;
 
